symengine/basic.h

```diff
@@ -158,6 +158,9 @@ class Basic : public EnableRCPFromThis<Basic> {
     //! Returns the list of arguments
     virtual vec_basic get_args() const = 0;
 
+    //! Returns the numerator/denominator form of the expression
+    virtual void as_numer_denom(const Ptr<RCP<const Basic>> &numer, const Ptr<RCP<const Basic>> &denom) const;
```

Comment:
We are trying to make everything non-member functions, but only keeping those that are already member functions. See https://github.com/symengine/symengine/pull/527#issue-94633819